For series use ordering_in_sect Pubd1311

myucekul opened the pull request:
The first change for series page is to use 'default' as default sorting. The second change is to utilize ordering_in_sect is available. The items with the order are placed first and then the remaining items.

hardyoyo commented:
Aside from a few minor issues (including the failed linter issue from Rubocop) I think this ought to work. I'm going to give it s spin this morning, and will make a PR to this PR to include the suggested changes.

myucekul commented:
> Aside from a few minor issues (including the failed linter issue from Rubocop) I think this ought to work. I'm going to give it s spin this morning, and will make a PR to this PR to include the suggested changes.

Thank you. I have incorporated variable change and removed debug puts.
